src/mcp_agent/mcp/mcp_aggregator.py

```
@@ -241,108 +272,156 @@
 
-            logger.debug("MCPAggregator created and initialized.")
-            return instance
-        except Exception as e:
-            logger.error(f"Error creating MCPAggregator: {e}")
-            await instance.__aexit__(None, None, None)
+                logger.debug("Loading servers...")
+                await instance.load_servers()
+
+                logger.debug("MCPAggregator created and initialized.")
+                return instance
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)